staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -184,6 +184,12 @@ const (
 	// Enables server-side field validation.
 	ServerSideFieldValidation featuregate.Feature = "ServerSideFieldValidation"
 
+	// owner: @enj
+	// Deprecated: v1.29
+	//
+	// When enabled, http2 DOS mitigations for unauthenticated clients are skipped.
```

Comment:
include the known reasons for using this in a comment (already-mitigating load balancer or network-private apiserver where the built-in mitigation impacts legitimate clients)